uninstall.php

```diff
@@ -13,16 +13,17 @@
 }
 
 /**
- * Deletes all poems from the database when the plugin is uninstalled.
+ * Deletes demo from the database when the plugin is uninstalled.
  */
 $args = array(
 	'post_type'      => 'poem',
-	'posts_per_page' => -1,
 	'post_status'    => 'any',
+	'title'          => 'Demo Poem: Windswept',
```

Comment:
This is now the second (or third) place where this important title is found.

Consider making a value object class that holds this and can be referenced where needed. Something like:

```php
class Demo_Poem {
	public static $post_title = 'Demo Poem: Windswept';
	public static $post_status = 'draft';
	// post type should be grabbed from elsewhere, since it needs to stay consistent with what's been filtered.
}
```

Then here, refer to it as a statically-accessed property:

```php
	'title' => Demo_Poem::$post_title,
```

The demo poem is a thing, an entity, so lets have all the knowledge we have about it live in one object.

Reply:
Thanks, this was fun to refactor!